HotelBooking.Model/Entities/Duration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using HotelBooking.Common.Base;
+using System.ComponentModel.DataAnnotations;
+
+namespace HotelBooking.Model.entity
```

Comment:
```suggestion
namespace HotelBooking.Model.Entity
```
should uppercase the first letter